scipy/spatial/transform/__init__.py

```diff
@@ -0,0 +1,28 @@
+"""
+========================================================
+Spatial Transformations (:mod:`scipy.spatial.transform`)
+========================================================
+
+.. currentmodule:: scipy.spatial.transform
+
+This package implements various spatial transformations. For now,
+only rotations are supported.
+
+Class Reference
```

Comment:
I think the section should be named as `Rotation in 3 dimensions` (or something like that). It will contain `Rotation`, `Slerp`, etc.